src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -180,16 +179,42 @@ public static String getIndividualFullName(Entity entity) {
         return entity.getName().getFullNameNoTitle();
     }
 
-    public static Optional<String> getIndividualPreferredHearingChannel(String hearingType, HearingSubtype hearingSubtype) {
-        if (hearingType == null || hearingSubtype == null) {
-            return Optional.empty();
+    public static String getIndividualPreferredHearingChannel(String hearingType,
+                                                                        HearingSubtype hearingSubtype,
+                                                                        HearingOptions hearingOptions) {
+        if (eitherNull(hearingType, hearingSubtype)) {
```

Comment:
Don't think you need this as a whole method. Re: your comment below Harry's fix has gone in so you'd be overwriting it